src/client/modules/my/app/app.html

```diff
@@ -1,5 +1,5 @@
 <template>
-    <h1>Matt's Component</h1>
+    <h1>Tanner's Component</h1>
```

Comment:
At some point, we should make an official header/banner/title, and call it DominionStats